crates/network-primitives/src/traits.rs

```diff
@@ -67,6 +67,18 @@ pub trait HeaderResponse {
 
     /// Blob fee for the next block (if EIP-4844 is supported)
     fn next_block_blob_fee(&self) -> Option<u128>;
+
+    /// Miner/Coinbase of the block
+    fn miner(&self) -> Address;
+
+    /// Gas limit of the block
+    fn gas(&self) -> u128;
+
+    /// Mix hash of the block
+    fn mix_hash(&self) -> Option<B256>;
```

Comment:
this needs docs, that is is now `prevrandao` the  randomness value for the block